training/src/anemoi/training/config/model/transformer.yaml

```diff
@@ -14,6 +14,7 @@ processor:
   num_heads: 16 # GraphTransformer or Transformer only
   window_size: 512
   dropout_p: 0.0 # GraphTransformer
+  attention_implementation: flash_attention # flash_attention, flex_attention, scaled_dot_product_attention
 
```

Comment:
Are `use_alibi_slopes` and `softcap` meant to also be passed via the config? Since in the default we have flash_attention I was wondering if it would be better to also include the defaults for those two other parameters